components/Pages/SearchPage/index.tsx

```diff
@@ -0,0 +1,77 @@
+import PopluarStocks from '@/components/PopularStocks';
+import RecentSearchWords from '@/components/RecentSearchWords';
+import SearchInput from '@/components/SearchInput';
+import SearchResults from '@/components/SearchResults';
+import BottomFixedButton from '@/components/common/Button/BottomFixedButton';
+import { useState } from 'react';
+
+function SearchStockPage() {
+  // 검색 결과 데이터
+  // const searchResults = [
+  //   { name: 'abca', tickercode: 'ABCA' },
+  //   { name: 'aaca', tickercode: 'AACA' },
+  //   { name: 'aaaa', tickercode: 'AAAA' },
+  //   { name: 'abbc', tickercode: 'ABBC' },
+  //   { name: 'abbb', tickercode: 'ABBB' },
+  //   { name: '가나다라', tickercode: '000001' },
+  //   { name: '가가다라', tickercode: '000011' },
+  //   { name: '가가가라', tickercode: '001122' },
+  //   { name: '가가가가', tickercode: '112233' },
+  //   { name: '가나나라', tickercode: '113344' },
+  // ];
+  // 최근 검색어 데이터
+  // const recentSearchWords = ['APPL', 'MSFT', 'JEPI'];
+
+  // const [pages, setPages] = useState('first');
+
+  // 2-2에서 2-3으로 넘어가기
+  const [value, setValue] = useState('');
+  const [isSearchActive, setIsSearchActive] = useState(false);
+  // const [value, setValue] = useState('');
+
+  const handleShowSearchResult = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setValue(e.target.value);
+  };
```

Comment:
```suggestion
  const handleShowSearchResult: ChangeEventHandler<HTMLInputElement> = (e) => {
    setValue(e.target.value);
  };
```
이렇게 변수명에다가 핸들러함수라고 타입정의하는 방법도 있습니다! 참고해보시고 마음에 드시는쪽으로 사용하시면 될 것 같아요